backend/src/schema/resolvers/socialMedia.spec.js

```diff
@@ -98,14 +98,19 @@ describe('SocialMedia', () => {
       const variables = {
         url: '',
       }
-      await expect(client.request(mutationC, variables)).rejects.toThrow('Input is not a URL')
+      await expect(client.request(mutationC, variables)).rejects.toThrow(
```

Comment:
I know you didn't add this, but I have no idea what `mutationC` is in this case
I would need to open up this file to see, which when reviewing a big `PR` makes it even more time-consuming
Not to harp on the same point, but if we get in the habit of instead naming it something generic like `mutation` or `mutationA` then I think we can help ourselves and others to read the code faster.
Or maybe others don't agree with me, in which case I can make an effort to deal with it and not continuously bring it up.